### ISSUE TITLE
Add more MAL integration options

### ISSUE THREAD

**Issue by SimonMattila** [lap00zza/9anime-Companion#95]
**Select your issue type:** (check at least one)
- [ ] Bug
- [ ] Question
- [x] Suggestion
- [ ] Other (please describe):

**Describe your issue:**
I would like the ability to change watch-status and score using the extension, just like the Tampermonkey/Greasemonkey script KissAnimelist does. Attached a screenshot plus a link to the script.
[Screenshot](http://prntscr.com/hd47ec)
[Script](https://greasyfork.org/en/scripts/27564-kissanimelist)

**Comment by lap00zza** [comment]
Yes this can be done. Actually 9ac already supports doing it, but just the UI is not implemented. This will be done soon.

**Comment by SimonMattila** [comment]
Not the best thing I could cook up but a mere illustration on how it could look like, a drop down menu for Status and rating then a popup asking at the first episode if you wish to set it as watching and one at the last episode (as per my comment in #68 regarding both #68 and #99 )

![setwatching](https://user-images.githubusercontent.com/10836780/33142443-c91214fa-cfb6-11e7-81b9-826b91b367f2.png)
